docs/docs/headless-cms.md

```diff
@@ -36,7 +36,7 @@ Here are more resources for guides, plugins, and starters for CMS systems you ca
 | [Contentstack](https://www.contentstack.com/) | [guide](/docs/sourcing-from-contentstack)                                        | [docs](/packages/gatsby-source-contentstack)         | [starter](/starters/contentstack/gatsby-starter-contentstack/)      |
 | [ButterCMS](https://buttercms.com/)           | [guide](/docs/sourcing-from-buttercms/)                                          | [docs](/packages/gatsby-source-buttercms)            | [starter](/starters/ButterCMS/gatsby-starter-buttercms/)            |
 | [Ghost](https://ghost.org/)                   | [guide](/docs/sourcing-from-ghost/)                                              | [docs](/packages/gatsby-source-ghost/)               | [starter](/starters/TryGhost/gatsby-starter-ghost/)                 |
-| [Kentico Cloud](https://kenticocloud.com/)    | [guide](/docs/sourcing-from-kentico-cloud)                                       | [docs](/packages/gatsby-source-kentico-cloud)        | [starter](/starters/Kentico/gatsby-starter-kentico-cloud/)          |
+| [Kentico Kontent](https://kontent.ai/)        | [guide](/docs/sourcing-from-kentico-kontent)                                     | [docs](/packages/gatsby-source-kontent)              | [starter](/starters/Kentico/gatsby-starter-kentico-cloud/)          |
```

Comment:
Do you mean `gatsby-starter-kontent`? 
```suggestion
| [Kentico Kontent](https://kontent.ai/)        | [guide](/docs/sourcing-from-kentico-kontent)                                     | [docs](/packages/gatsby-source-kontent)              | [starter](/starters/Kentico/gatsby-starter-kontent/)          |
```

Reply:
Yes. I made the initial change before the starter was renamed and then overlooked this change after. Thanks!